current-federal-non-dotgov.csv

```diff
@@ -2063,3 +2064,4 @@ zipstation.net,Federal Agency - Executive,United States Postal Service,,Washingt
 zip-station.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+
```

Comment:
Remove blank line to pass linter tests:
```suggestion
```

Reply:
Resolved via https://github.com/cisagov/scan-target-data/pull/103/commits/0a8088997fbf5cf9259b375f65472457d086e35f.